app/src/main/java/org/mozilla/reference/browser/settings/SettingsActivity.kt

```diff
@@ -9,12 +9,17 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.MenuItem
 import mozilla.components.support.base.feature.UserInteractionHandler
+import org.mozilla.reference.browser.R.color.navigationBarColor
+import org.mozilla.reference.browser.R.color.statusBarColor
+import org.mozilla.reference.browser.ext.setWindowTheme
 
 class SettingsActivity : AppCompatActivity(), SettingsFragment.ActionBarUpdater {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        setWindowTheme(statusBarColor, navigationBarColor)
```

Comment:
This should also be made with theming and not in code

Reply:
I have through the theme but the status bar and navigation bar buttons/text do not change the colour to black on white theme. This image has the status bar and navigation bar to it.



![Screenshot_1580311432](https://user-images.githubusercontent.com/6861614/73370139-05f96b80-42b4-11ea-94e6-46a63f604f15.png)


I can do this to fix this but this required min API 27 and 23 resp.
```
        <item name="android:windowLightNavigationBar">true</item>
        <item name="android:windowLightStatusBar">true</item>
```